exporter/containerimage/export.go

```diff
@@ -161,6 +166,9 @@ func (e *imageExporter) Resolve(ctx context.Context, opt map[string]string) (exp
 			i.meta[k] = []byte(v)
 		}
 	}
+	if esgz && !i.ociTypes {
+		logrus.Warn("estargz compression should be used with oci-mediatypes for enabling layer verification")
```

Comment:
You don't want to make this an error? Or always imply ocitypes for stargz, at least if not set?

Reply:
Fixed this to always imply ocitypes with a warning log.